### PR TITLE
Minor fix to SoundRecorder documentation

### PR DESCRIPTION
**Pull request by halatmit**

### CODE REVIEW

**Comment by jisqyv** [comment]
@halatmit LGTM

File: appinventor/components/src/com/google/appinventor/components/runtime/SoundRecorder.java [hunk 1]
```diff
@@ -119,9 +119,11 @@ public SoundRecorder(final ComponentContainer container) {
    * @return  savedRecording path to recording
    */
   @SimpleProperty(
-      description = "Specifies the path to the file where the recording is stored. " +
+      description = "Specifies the path to the file where the recording should be stored. " +
           "If this proprety is the empty string, then starting a recording will create a file in " +
-          "an appropriate location.",
+          "an appropriate location.  If the property is not the emppty string, it should specify " +
```
Comment (jisqyv) [inline appinventor/components/src/com/google/appinventor/components/runtime/SoundRecorder.java]: Ooops emppty spelled wrong, I'll fix it on Gerrit

File: appinventor/docs/reference/components/media.html [hunk 2]
```diff
@@ -651,8 +651,18 @@ <h2 id="Sound">Sound</h2>
...
-                    <h3> Properties </h3>
-                    none
+		    <h3>Properties</h3>
+		    <dl>
+		      <dt><code>SavedRecording</code></dt>
+		      <dd>Specifies the path to the file where the
+		      recording should be stored. If this proprety is
+		      the empty string, then starting a recording will
+		      create a file in an appropriate location.  If
+		      the property is not the emppty string, it should
```
Comment (jisqyv) [inline appinventor/docs/reference/components/media.html]: Here too... :-)

Comment (halatmit) [inline appinventor/docs/reference/components/media.html]: thanks :)



== Hal Abelson
hal@mit.edu
Prof. of Comp. Sci. and Eng.
MIT Dept. of Elec. Eng. and Comp. Sci.
MIT Media Lab, Second Class Faculty


On Mon, Aug 10, 2015 at 11:01 AM, Jeffrey Schiller <notifications@github.com

**Comment by jisqyv** [comment]
MERGED

**Pull request closed without merging**